config/locales/en.yml

```diff
@@ -68,6 +68,7 @@ en:
     confirm_delete: Are you sure you want to delete this user?
     service_provider_deleted: Success! You have deleted %{issuer}
     service_provider_saved: Success! You have saved %{issuer}
+    service_provider_iaa_notice: Please note that you will need a signed Inter-Agency Agreement with login.gov in order to go live in production. Visit <a href="https://developers.login.gov/production/" target="_blank">https://developers.login.gov/production</a> for more details or email <a href="mailto:partners@login.gov">partners@login.gov</a> with any questions.
```

Comment:
If you suffix this key with `_html`, it will automatically be marked as HTMLsafe by the i18n gem

Reply:
oooh neat trick!

Reply:
This didn't work 😕 - I think we need to add `sanitize` (based on [this thread](https://stackoverflow.com/questions/17375232/rails-i18n-html-suffix-rule-and-translate-helper-called-from-a-controller) - it's old but makes sense since we're calling it in the controller).